<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScrabbleGui</class>
 <widget class="QMainWindow" name="ScrabbleGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>493</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>493</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>493</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Scrabble Solver</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTableWidget" name="board">
      <property name="minimumSize">
       <size>
        <width>481</width>
        <height>477</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>481</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>30</number>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>30</number>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>30</number>
      </attribute>
      <row>
       <property name="text">
        <string>1</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>2</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>3</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>4</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>5</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>6</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>7</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>8</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>9</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>10</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>11</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>12</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>13</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>14</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>15</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>3</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>4</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>5</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>6</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>7</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>8</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>9</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>10</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>11</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>12</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>13</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>14</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>15</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="letters">
        <property name="text">
         <string/>
        </property>
        <property name="maxLength">
         <number>7</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="solve_btn">
        <property name="text">
         <string>Solve</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="new_btn">
          <property name="text">
           <string>New</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="save_btn">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="load_btn">
          <property name="text">
           <string>Load</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="results"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <action name="newScrabble">
   <property name="text">
    <string>Scrabble</string>
   </property>
  </action>
  <action name="newWordsWithFriends">
   <property name="text">
    <string>Words With Friends</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>solve_btn</sender>
   <signal>clicked()</signal>
   <receiver>ScrabbleGui</receiver>
   <slot>solve()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>733</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>490</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>results</sender>
   <signal>itemPressed(QListWidgetItem*)</signal>
   <receiver>ScrabbleGui</receiver>
   <slot>highlight(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>733</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>490</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>save_btn</sender>
   <signal>clicked()</signal>
   <receiver>ScrabbleGui</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>733</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>490</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>load_btn</sender>
   <signal>clicked()</signal>
   <receiver>ScrabbleGui</receiver>
   <slot>load()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>894</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>490</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>solve()</slot>
  <slot>save()</slot>
  <slot>load()</slot>
  <slot>highlight(QListWidgetItem*)</slot>
 </slots>
</ui>
